app/frontend/src/features/profile/view/leaveReference/formSteps/Appropriate.tsx

```diff
@@ -54,10 +55,10 @@ export default function Appropriate({
     setReferenceValues(values);
     referenceType === referenceTypeRoute[ReferenceType.REFERENCE_TYPE_FRIEND]
       ? history.push(
-          `${leaveReferenceBaseRoute}/${referenceType}/${user.userId}/rating`
+          `${leaveReferenceBaseRoute}/${referenceType}/${user.userId}/${steps[1]}`
         )
       : history.push(
-          `${leaveReferenceBaseRoute}/${referenceType}/${user.userId}/${hostRequest}/rating`
+          `${leaveReferenceBaseRoute}/${referenceType}/${user.userId}/${hostRequest}/${steps[1]}`
```

Comment:
```suggestion
          `${leaveReferenceBaseRoute}/${referenceType}/${user.userId}/${hostRequestId}/${steps[1]}`
```
I think you'd have to update this (and on line 44, can't comment on unchanged lines directly!) since the rename in `routes.ts`